packages/react-components/src/components/CaptionsSettingsModal.tsx

```diff
@@ -347,7 +347,9 @@ export const CaptionsSettingsModal = (props: CaptionsSettingsModalProps): JSX.El
           styles={CaptionsSettingsModalStyle}
         >
           <Stack horizontal horizontalAlign="space-between" verticalAlign="center" className={titleContainerClassName}>
-            <Text className={titleClassName}>{strings?.captionsSettingsModalTitle}</Text>
+            <Text className={titleClassName} role="heading">
```

Comment:
This might need to have `aria-level` as well
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-level#with_heading_role

![image](https://github.com/user-attachments/assets/cf40bd54-8cda-4973-8d05-9cde51790681)

Reply:
Good catch, this will need a aria-level @carocao-msft

Reply:
Updated